src/main/java/seedu/address/model/person/Person.java

```diff
@@ -124,8 +126,10 @@ public String toString() {
             builder.append("; Tags: ");
             tags.forEach(builder::append);
         }
-        builder.append("; Birthday: ")
-                .append(getBirthday());
+        if (getBirthday().isPresent()) {
+            builder.append("; Birthday: ")
+                    .append(getBirthday().map(Birthday::toString).get());
```

Comment:
I don't think need to map to toString here, since appending the Birthday to a string will automatically return the toString. `getBirthday().get()` should suffice.